modules/alloy/src/main/ts/ephox/alloy/behaviour/tooltipping/TooltippingSchema.ts

```diff
@@ -14,7 +14,7 @@ export default [
   FieldSchema.defaulted('tooltipComponents', []),
   FieldSchema.defaultedFunction('delayForShow', Fun.constant(300)),
   FieldSchema.defaultedFunction('delayForHide', Fun.constant(300)),
-  FieldSchema.defaultedStringEnum('mode', 'normal', [ 'normal', 'follow-highlight' ]),
+  FieldSchema.defaultedStringEnum('mode', 'normal', [ 'normal', 'follow-highlight', 'children-keyboard-focus' ]),
```

Comment:
Should this just be `children-focus` given it works for both keyboard and mouse focus?

Reply:
As discussed (leaving a note here), as the collection shifts the focus directly on `mouseover`, we can only listen to `focusin` and `postblur` events in the tooltipping config, and show tooltip anchored to the focussed element. There will be be another `mode` in an upcoming PR that runs on both on mouse and keyboard when a child element inside a component receives focus or is being hovered.